### ISSUE TITLE
Extension does not work

### ISSUE THREAD

**Issue by Lunixee** [Swiftzerr/kahoot-ai-extension#1]
The extension does not work, it doesn't show the answers for the questions.

**Comment by elliotjarnit** [comment]
Try using the extension with the answers shown on the screen, is it any different?

**Comment by Lunixee** [comment]
Nope still does not work ¯\_(ツ)_/¯

**Comment by elliotjarnit** [comment]
Could you send a screenshot of your console? This project is abandoned but I could fix it again.

**Comment by Lunixee** [comment]
what console?

**Comment by elliotjarnit** [comment]
The developer console on the Kahoot page. Just right click and go into inspect.

**Comment by Lunixee** [comment]
![image](https://user-images.githubusercontent.com/115111841/202560727-65518c2c-eff8-47aa-91b3-00da1401b3ca.png)

**Comment by Lunixee** [comment]
Yeah I checked the console when I first installed the extension but it just showed the "Could not load content" error and the "Too Many Requests" error. I think the source code is just outdated tbh

**Comment by elliotjarnit** [comment]
Ill try and fix it soon and will let you know when i do. Thanks for reporting this.

**Comment by elliotjarnit** [comment]
I just tested out the extension and it seems to work fine. Could you send me a link to the kahoot you are testing this with?

**Comment by LeapToken** [comment]
ok

**Comment by Lunixee** [comment]
Honestly, I've tested this kahoot AI on every type of kahoot and it didn't work (Kept showing me errors)
